src/main/java/be/acara/events/controller/UserController.java

```diff
@@ -50,4 +51,21 @@ public void signUp(@RequestBody User user) {
         boolean check = userService.checkUsername(username);
         return ResponseEntity.ok(check);
     }
+
+    @GetMapping("/{id}/likes")
+    public ResponseEntity<Boolean> doesUserLikeThisEvent(@PathVariable("id") Long id) {
+        return ResponseEntity.ok(!userService.doesUserLikeThisEvent(id));
```

Comment:
Why is this inverted with an "!"?

Reply:
id here implies userId, not eventId, but it's being used as the event id.
This is both unclear from an API design standpoint and from a variable naming standpoint.

The correct API endpoint should be `/api/users/{userId}/events/{eventId}/.

I understand that you can get the user's id from the logged in user, but that's an implementation detail that should not be enforced in your API design.
For instance: what if you want to extend your application, so you have public profile pages, where you want to show a user's liked events? You would have to change your entire API just to account for this small change, while with proper API design, it would be a matter of changing permissions.

It might also be a good idea to add this like-information to the EventDto so you don't have to do an extra call for each event